scripts/dataproc-workflow-composer/clouddq_composer_dataplex_task_job.py

```diff
@@ -156,6 +156,20 @@ def _get_dataplex_job_state() -> str:
         print(f"CloudDQ task status is {task_status}")
     return task_status
 
+def _get_dataplex_task() -> str:
+    """
+    This method will return the status for the task.
+    Args:
+    Returns: str
+    """
+    headers = get_session_headers()
+    res = requests.get(
+        f"{DATAPLEX_ENDPOINT}/v1/projects/{DATAPLEX_PROJECT_ID}/locations/{DATAPLEX_REGION}/lakes/{DATAPLEX_LAKE_ID}/tasks/{DATAPLEX_TASK_ID}",
+        headers=headers)
+    if res.status_code == 200:
+        return "task_exist"
+    else:
```

Comment:
can we instead please check res.status_code == 404 for task not exist, and in the "else" branch return an error?

Reply:
I have incorporated the suggestion.